src/rgw/driver/rados/rgw_reshard.cc

```diff
@@ -1652,7 +1652,7 @@ int RGWReshard::process_entry(const cls_rgw_reshard_entry& entry,
     // needed to perform the calculation before calling
     // calculating_preferred_shards() in this class
     store->getRados()->calculate_preferred_shards(
-      dpp, num_entries, current_shard_count,
+      dpp, bucket_info, num_entries, current_shard_count,
```

Comment:
i guess this needs to be adopted as well.

Reply:
ty!